modules/mcuboot/CMakeLists.txt

```
@@ -203,6 +203,14 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
       )
   endif()
 
+  # Temporary hacky solution to enable USB CDC node in mcuboot for NCS builds
```

I don't think we should use such wording in our build code.

Better to just state why this is done, and if there is a task to improve it, then refer to that task.
Otherwise just refer to a generic cleanup.

For example:
```suggestion
  # For devices with USB then a USB CDC node in mcuboot is required.
  # As USB CDC is generally needed for this mcuboot use-case it should be considered if this solution could be better generalized.
```

---

Will reword the comment